cern.root.ROOT.yaml

```
@@ -0,0 +1,27 @@
+app-id: cern.root.ROOT
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: root
+rename-desktop-file: root
```

it's "destkop-file" so:
```suggestion
rename-desktop-file: root.desktop
```

note that your local build you acknowledged having done would have failed too.

---

In fact I changed that at the last minute. The compilation I did did not finish because my system crashed, according to me due to lack of ram.